packages/orbit-components/src/Button/Button.stories.tsx

```diff
@@ -43,6 +43,17 @@ export const BasicButtons = () => {
   );
 };
 
+export const ButtonInStackWithLimitedWidth = () => {
+  return (
+    <div style={{ width: 300 }}>
+      <Stack direction="row" align="stretch">
+        I am a llama and I like to llama
+        <Button>To be or not to be</Button>
+      </Stack>
+    </div>
+  );
+};
+
```

Comment:
Instead of this story, this could be just a visual test. I don't think it's that interesting for a visual story, as it serves solely for us to be sure it does not wrap. Non-blocking, but I would remove the story and instead add a visual test. WDYT?

Reply:
Idk, there are a bunch of other tests that needed to be added then, but can be solved in the future